src/main/webapp/sass/swc.scss

```diff
@@ -34,12 +34,13 @@ a {color:$synapse-blue;text-decoration:none;}
 a:visited{color:$synapse-blue; text-decoration:none;}
 a:hover{color:$synapse-blue; text-decoration:underline;}
 a:focus{color:$synapse-blue; text-decoration:underline;}
```

Comment:
Lightened up color value when elements were not enabled (to make it more obvious in the UI).  Also apply to checkbox and radio "disabled" state.